src/Microsoft.VisualStudio.Threading.Analyzers.Tests/MultiAnalyzerTests.cs

```diff
@@ -171,7 +172,7 @@ public class A {
         E().ToString();
         E()();
         string v = nameof(E);
-        return null;
+        {|VSTHRD112:return null;|}
```

Comment:
Instead of introducing diagnostics to this test, can we just update the test to `return Task.CompletedTask;`